modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -83,7 +84,14 @@ export default class AdvancedOpts {
             .append( 'button' )
             .classed( 'advanced-opts-reset button secondary strong', true )
             .text( 'Reset' )
-            .on( 'click', () => this.createGroups() );
+            .on( 'click', () => {
+                let hiddenOpts = [];
+                let getOpts = d3.selectAll('.group-body.fill-white.hidden').data();
```

Comment:
I think here you can just use that d3 `.each` function, and in that callback just push the .name property. Rather than making a new list out of the selection and looping over it